runtime/compiler/control/rossa.h

```diff
@@ -86,6 +86,7 @@ typedef enum {
    compilationStreamVersionIncompatible            = compilationFirstJITServerFailure+3,
    compilationStreamInterrupted                    = compilationFirstJITServerFailure+4,
 #endif /* defined(J9VM_OPT_JITSERVER) */
+   compilationAotHasInvokeSpecialInterface = 60,
```

Comment:
This is going to be harder to maintain if we add or delete JITServer related failures. Maybe add a `compilationLastJITServerFailure` and start counting from there.
Even better, don't add any number; let the compiler do the counting. Just add a comment with the supposed value.